make-install.js

```diff
@@ -1,12 +1,22 @@
 const generateInstallScript = ({
+  email,
+  multiSite,
   wpConfig,
   installPlugins,
   activatePlugins,
 }) => {
+  const adminEmail = email || 'admin@example.com';
+  let url = "$WP_HOST_NAME";
+  if (wpConfig.WP_PORT !== 80) {
+    url += ":$WP_PORT";
+  }
+  const installMethod = multiSite ? 'multisite-install' : 'install';
   return `
+#!/usr/bin/env bash
+
 # wait for the wordpress and db containers to run  
-sleep 20;
-wp core install --path="/var/www/html" --url="$WP_HOST_NAME:$WP_PORT" --title="WooCommerce Dev" --admin_user=admin --admin_password=password --admin_email=admin@example.com;
+sleep 10;
```

Comment:
I didn't try removing this, but it might not be necessary anymore now that we mount the plugins directory as well.